<launch>
	<node pkg="marker_localization" type="marker_generate_boards" name="marker_generate_boards" clear_params="true" output="screen">
		<param name="output_directory" value="/tmp/" type="str"/>

		<rosparam command="load" file="$(find marker_localization)/launch/board_config.yaml"/>
	</node>
</launch>
